### PR TITLE
Updates for FastStart pre-work.

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: workshop/pre-work/README.md [hunk 1]
```diff
@@ -3,41 +3,32 @@
 Before we get started, we will download some assets and complete some setup for our workshop. This section is broken up into the following steps:
 
 - [Pre-work](#pre-work)
-  - [1. Download or Clone the Repository](#1-download-or-clone-the-repository)
+  - [1. Download the data](#1-download-the-data)
```
Comment (omidmeh) [inline workshop/pre-work/README.md:6]: `Data` should be in Upper case for consistency? (They are all Camel

File: workshop/pre-work/README.md [hunk 2]
```diff
@@ -217,41 +170,22 @@ Alternatively, you can use the IBM Cloud Console to generate the IBM Cloud API k
 
 This is the *api_key* value that you will need to provide in your Jupyter notebooks for accessing the Watson Machine Learning service instance.
 
-### Get the Watson Machine Learning service instance location
-
-<!-- TODO: In case we can only use Frankfurt and Dallas, remove the rest. -->
-**Option 1:**
```
Comment (omidmeh) [inline workshop/pre-work/README.md:223]: They need to fill this in for deploying to the right place in the notebook iirc. Can you check where we fill this in the notebook? I can't remember off the top of my head.

**Review by omidmeh — CHANGES_REQUESTED** [review]
Added a few comments

File: workshop/pre-work/README.md (hunk 2 above)
Comment (scottdangelo) [reply workshop/pre-work/README.md:223]: In the `machine-learning-in-jupyter-notebook`. We aren't doing that module.

File: workshop/pre-work/README.md (hunk 1 above)
Comment (scottdangelo) [reply workshop/pre-work/README.md]: done

File: workshop/pre-work/README.md (hunk 2 above)
Comment (omidmeh) [reply workshop/pre-work/README.md:223]: Ah right! cool.

**Review by omidmeh — APPROVED** [review]

**Pull request merged**